facebookads/api.py

```diff
@@ -237,7 +237,19 @@ def call(self, method, path, params=None, headers=None, files=None):
 
         self._num_requests_attempted += 1
 
-        if not isinstance(path, six.string_types):
+        batch = params.get('batch')
+        relative_url = None
+
+        if batch and isinstance(batch, list):
+            relative_url = batch[0].get('relative_url')
```

Comment:
Does this mean that if the first entry in a batch has a `relative_url` set, other operations in the batch (which might be for completely different actions) will also have the endpoint overridden?